tests/test_kernels.py

```diff
@@ -380,32 +364,38 @@ def test_ard_init_scalar(D):
     lengthscale or a suitable array of lengthscale
     """
     kernel_1 = gpflow.kernels.SquaredExponential(lengthscale=2.3)
-    kernel_2 = gpflow.kernels.SquaredExponential(lengthscale=np.ones(D) * 2.3, ard=True)
+    kernel_2 = gpflow.kernels.SquaredExponential(lengthscale=np.ones(D) * 2.3)
     lengthscale_1 = kernel_1.lengthscale.read_value()
     lengthscale_2 = kernel_2.lengthscale.read_value()
     assert np.allclose(lengthscale_1, lengthscale_2, atol=1e-10)
 
 
-@pytest.mark.parametrize('N', [4, 7])
-@pytest.mark.parametrize('ard', [True, False, None])
-def test_ard_init_shapes(N, ard):
-    with pytest.raises(tf.errors.InvalidArgumentError):
-        k1 = gpflow.kernels.SquaredExponential(lengthscale=np.ones(2), ard=ard)
-        k1(rng.randn(N, 4))
-    with pytest.raises(tf.errors.InvalidArgumentError):
-        k2 = gpflow.kernels.SquaredExponential(lengthscale=np.ones(3), ard=ard)
-        k2(rng.randn(N, 2))
-
-
-@pytest.mark.parametrize('D', [4, 7])
-def test_ard_init_MLP(D):
-    """
-    For ard kernels, make sure that kernels can be instantiated with a single
-    lengthscale or a suitable array of lengthscale
-    """
-    kernel_1 = gpflow.kernels.ArcCosine(weight_variances=1.23, ard=True)
-    kernel_2 = gpflow.kernels.ArcCosine(weight_variances=np.ones(3) * 1.23,
-                                        ard=True)
-    variances_1 = kernel_1.weight_variances.read_value()
-    variances_2 = kernel_2.weight_variances.read_value()
-    assert np.allclose(variances_1, variances_2, atol=1e-10)
+def test_ard_invalid_active_dims():
+    msg = r"Dimension of `active_dims` \[1\] does not match dimension of the ard parameter \(2\)"
+    with pytest.raises(ValueError, match=msg):
+        gpflow.kernels.SquaredExponential(lengthscale=np.ones(2), active_dims=[1])
+
+
+#@pytest.mark.parametrize('N', [4, 7])
```

Comment:
Why this part is commented out?

Reply:
This is the associated test for the aforementioned commented code.